packages/grafana-data/src/events/types.ts

```diff
@@ -54,12 +54,17 @@ export interface BusEventHandler<T extends BusEvent> {
   (event: T): void;
 }
 
+export enum EventFilter {
```

Comment:
Could this be changed to an object that has predefined functions for each filter type? Then we could provide a function that would white list events that plugins are allowed to publish to the "global" event system of Grafana.